engines/osxnative/init_darwin.go

```diff
@@ -0,0 +1,8 @@
+package osxnative
+
+import "github.com/taskcluster/taskcluster-worker/engines/extpoints"
+
+func init() {
+	// Register the mac engine as an import side-effect
+	extpoints.EngineProviders.Register(engineProvider{}, "macosx")
+}
```

Comment:
because the file is darwin only... the other files can compile on all platforms... at least for now...

@walac should totally overwrite what I did here when we merge his patch (hopefully soon too)...